elytron/src/main/java/org/wildfly/extension/elytron/AuthenticationClientDefinitions.java

```diff
@@ -505,6 +507,18 @@ static ResourceDefinition getAuthenticationContextDefinition() {
                 return () -> finalContext.apply(parentSupplier.get());
             }
 
+            @Override
+            protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model) throws OperationFailedException {
+                commonRequirements(installService(context, model)).setInitialMode(ServiceController.Mode.ACTIVE).install();
+            }
+
+            ServiceBuilder<AuthenticationContext> installService(OperationContext context, ModelNode model)  throws OperationFailedException {
+                ServiceTarget serviceTarget = context.getServiceTarget();
+                ServiceBuilder<AuthenticationContext> serviceBuilder =
+                        (ServiceBuilder<AuthenticationContext>)context.getCapabilityServiceTarget().addCapability(AUTHENTICATION_CONTEXT_RUNTIME_CAPABILITY);
+                TrivialService<AuthenticationContext> certificateAuthorityTrivialService = new TrivialService<AuthenticationContext>(getValueSupplier(serviceBuilder, context, model));
```

Comment:
Very minor but looks like `certificateAuthorityTrivialService` should be renamed.

Reply:
Fixed. Thanks!